### ISSUE TITLE
.r is not working

### ISSUE THREAD

**Issue by wsmchael** [DWilliames/google-sheets-sync-figma#25]
These days, I use .r to rename layer,  but when I runing the plugin, it's not working.Nothing change.

**Comment by prakashkarkee** [comment]
yeah i agree with your problem beside there are `.n` , `.x `and `.i ` which works well. Lets hope `.r` problem will be resolved in future update by the developer @DWilliames  of the plugin.

**Comment by alivekenny** [comment]
Have this problem too. Is there any time estimation for fixing this bug?
